tools/stats/upload_binary_size_to_scuba.py

```diff
@@ -22,12 +24,11 @@ def get_size(file_dir):
         return 0
 
 
-def build_message(size):
-    pkg_type, py_ver, cu_ver, *_ = os.environ.get("BUILD_ENVIRONMENT", "").split() + [
-        None,
-        None,
-        None,
-    ]
+def build_message(size: int) -> Dict[str, Any]:
+    if "BUILD_ENVIRONMENT" in os.environ.keys():
+        pkg_type, py_ver, cu_ver, *_ = os.environ.get("BUILD_ENVIRONMENT", "").split()
+    else:
+        pkg_type, py_ver, cu_ver = "", "", ""
```

Comment:
This doesn't seem to match up with what the original code meant; could you explain why this is ok?

Reply:
```suggestion
    if "BUILD_ENVIRONMENT" in os.environ.keys():
        pkg_type, py_ver, cu_ver, *_ = os.environ.get("BUILD_ENVIRONMENT", "").split()
    else:
        pkg_type, py_ver, cu_ver = (None, None, None)
```

Reply:
hmm. you are right. I made this change b/c mypy complains concatenate string and None.

Reply:
if you want to satisfy `mypy`'s pickiness then you could do this instead:
```suggestion
    from typing import Optional, Sequence
    split_env: Sequence[Optional[str]] = os.environ.get("BUILD_ENVIRONMENT", "").split()
    pkg_type, py_ver, cu_ver, *_ = list(split_env) + [None, None, None]
```

Reply:
```suggestion
    build_env_split: List[Any] = os.environ.get("BUILD_ENVIRONMENT", "").split()
    pkg_type, py_ver, cu_ver, *_ = build_env_split + [None, None, None]
```